cli/crates/cli/src/deploy.rs

```diff
@@ -1,10 +1,45 @@
-use crate::{errors::CliError, output::report};
+use crate::{create::create_impl, errors::CliError, link::link_impl, output::report, prompts::handle_inquire_error};
 use backend::api::deploy;
+use common::{consts::PROJECT_METADATA_FILE, environment::Project};
+use inquire::Select;
+use strum::{Display, VariantArray};
+
+#[derive(VariantArray, Display, Clone)]
+enum UnlinkedDeploymentMethod {
+    Link,
+    Create,
+}
 
 #[tokio::main]
 pub async fn deploy() -> Result<(), CliError> {
+    let project = Project::get();
+
+    let project_metadata_file_path = project.dot_grafbase_directory_path.join(PROJECT_METADATA_FILE);
+
+    match project_metadata_file_path.try_exists() {
```

Comment:
I'd try it as such:

```rust
let project_metadata_file_exists = project_metadata_file_path.try_exists().map_err(CliError::ReadProjectMetadataFile)?;
if !project_metadata_file_exists {
  …
}
```

Reply:
I'll make a note of these two changes and add them in another PR as we want to release today